gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GenericStoreChangeEvent.avsc

```diff
@@ -9,9 +9,14 @@
     "doc" : "Primary key for the store",
     "compliance" : "NONE"
   }, {
-    "name" : "timestamp",
+    "name" : "txId",
```

Comment:
This seems to be an incompatible change in schema. Maybe add another field as txId and keep timestamp here?

Reply:
I want to rename this field to be more clear, this timestamp will be produceTimestamp specifically. I will update depending code in other places before deploying, is there still an issue with  incompatible schema change?

Reply:
Yes this is incompatible but I'll be changing the producer side in lockstep by updating both versions in our deployed service at once. I could add the new field at the end of the record but I did want to rename the timestamp field to be more clear about which timestamp it is now that I understand it was not the one I intended.